types/evidence.go

```diff
@@ -917,6 +917,138 @@ func (e PotentialAmnesiaEvidence) String() string {
 	return fmt.Sprintf("PotentialAmnesiaEvidence{VoteA: %v, VoteB: %v}", e.VoteA, e.VoteB)
 }
 
+type ProofOfLockChange struct {
+	Votes  []Vote        `json:"votes"`
+	PubKey crypto.PubKey `json:"pubkey"`
```

Comment:
can't we use `CommitSig` here?

Reply:
In replacement of the pubkey or the entire polc. I don't think you can use it as the entire polc and I don't understand why you would want it to replace the pubkey

Reply:
in replacement of votes. votes carry a lot of extra information, which may not be needed here

Reply:
Okay but you can change lock when you get 2/3+ prevotes. It doesn't have to be 2/3+ precommits (I'm guessing that's what a commitsig refers to)

Reply:
Right, okay